### DISCUSSION TITLE
_remotely-save-metadata-on-remote.json file in root of Obsidian Vault. How to not see this?

### DISCUSSION THREAD

**Discussion by j-peeters** [remotely-save/remotely-save#483] [Q&A]
Hi, Thanks for your great plugin. I have one problem though. On Windows 11 I see a _remotely-save-metadata-on-remote.json file in root of Obsidian Vault. I've disabled synching of files with a _ before it, but it still shows. 
On iPad and Android it is interestingly anough not visible. 
Is there a way to hide this. Thanks for any help!

**Comment by j-peeters** [comment]
I found to solution to hide it by deactivating the 'Detect all file extensions' setting in Obsidian Settings. It would still be great if I could find a way so that this file would not show up in my file tree. I'd love Obsidian to show also .pptx and other file types, which it doesn't with this setting off. Thanks.

**Comment by fyears** [comment]
the file is not created locally but created and uploaded to remote service. 

if you see local file _remotely-save-metadata-on-remote.json please just delete it

**Comment by j-peeters** [comment]
Hi @fyears Thanks for your reply. I deleted it multiple times but it comes back everytime. What could be the reason for this?

**Reply by fyears** [reply]
are you mixing any other sync methods?

**Comment by fyears** [comment]
new version doesn't have this file
